### ISSUE TITLE
GraphQL request adding an additional field `operationName` to curl and generateCode

### ISSUE THREAD

**Issue by mchrist13** [getinsomnia/insomnia#5694]
### Expected Behavior

When I generate a request or preview, I expect the GraphQL request to be made exactly how it appears

### Actual Behavior

GraphQL request appears to be modified and adds an additional field of `operationName`
Can be seen from generate preview or code:

e.g. 
` curl --request POST \
  --url REDACTED \
  --header 'Content-Type: application/json' \
  --header 'authorization: Bearer REDACTED \
  --data '{"query":"query {\n\tgetUserProfile {\n\t\tid\n\t\temail\n\t\tfirstName\n\t\tlastName\n\t\tEULA\n\t\tdisplayName\n\t}\n}\n","operationName":""}'
{"errors":[{"message":"Unknown operation named \"\"."}]}`

### Reproduction Steps

1. Modify an existing GraphQL query or mutation
2. Make request or generate code example

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.7.2

### What operating system are you using?

macOS

### Operating System Version

13.0.1

### Installation method

Download from Insomnia

**Comment by filfreire** [comment]
Can reproduce this, example generated curl:
```
curl --request POST \
  --url https://rickandmortyapi.com/graphql \
  --header 'Content-Type: application/json' \
  --data '{"query":"query ($id: ID!) {\n  character(id: $id) {\n    id\n    name\n    status\n    species\n    type\n    gender\n\t}\n}","variables":{"id":1},"operationName":""}'
  ```
  
  For the following query:
  ```
  query ($id: ID!) {
  character(id: $id) {
    id
    name
    status
    species
    type
    gender
	}
}
  ```

**Comment by filfreire** [comment]
@mchrist13 this should be fixed in the [latest 2022.7.5 release](https://github.com/Kong/insomnia/releases/tag/core%402022.7.5)

Examples:

![image](https://user-images.githubusercontent.com/11976836/213691620-67d876f7-b445-4e91-8492-8a5a3f7ec06b.png)
![image](https://user-images.githubusercontent.com/11976836/213691661-1f061a16-45ef-4481-a6f5-f290ad2f5a46.png)
